src/main/java/Duke.java

```diff
@@ -1,10 +1,122 @@
+import java.util.Scanner;
+import java.util.List;
+import java.util.ArrayList;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    public static void main(String[] args) throws DukeException {
+        System.out.println("Hello I'm Duke.");
+        System.out.println("What can I do for you?");
+
+        Scanner sc = new Scanner(System.in);
+
+        //create list
+        List<Task> list = new ArrayList<Task>();
+
+        while (true) {
+            String command = sc.nextLine();
+            String arr[] = command.split(" ", 2);
+            String firstWord = arr[0];
+
+            try {
+                if (command.equals("bye")) {
+                    System.out.println("Bye. Hope to see you again soon!");
+                    break;
+                } else if (command.equals("list")) {
+                    System.out.println("Here are the tasks in your list:");
+                    if (!list.isEmpty()) {
```

Comment:
Redundant checking. If the list is indeed empty, the for loop inside wouldn't run at all.